### PR TITLE
Improved message

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Thank your for the pull.  I improved the message to indicate that the data is not imported.

### CODE REVIEW

**Comment by bershanskiy** [comment]
I don't really see how this message is better.

**Comment by Gusted** [comment]
It should hopefully be explanatory that the data was not imported? Any error message should hopefully give that indication to the user.

**Comment by Gitoffthelawn** [comment]
Without making it clear, the user is left wondering: did it import partially?  did it import with errors?  did it not import at all?  The edit clears up any confusion.

File: src/ui/popup/manage-settings-page/import-settings.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default function ImportButton(props: ViewProps) {
 
     const dialog = context && context.store.isDialogVisible ? (
         <MessageBox
-            caption="The given file has incorrect JSON."
+            caption="Invalid JSON in import file.  Data not imported."
```
Comment (Gusted) [inline src/ui/popup/manage-settings-page/import-settings.tsx:49]:
```suggestion
            caption="Invalid JSON in import file. No data was imported."
```

**Review by Gusted — COMMENTED** [review]
Let's at least clear it up then.

File: src/ui/popup/manage-settings-page/import-settings.tsx (hunk 1 above)
Comment (Gitoffthelawn) [reply src/ui/popup/manage-settings-page/import-settings.tsx:49]: Sure

**Comment by bershanskiy** [comment]
I think settings import could use some work, but changing just the message text isn't particularly useful.

**Pull request closed without merging**